install.sh

```
@@ -0,0 +1,81 @@
+    echo "✅ Docker is already installed."
+fi
+
+# -------------------------------
+# 🔍 Install Trivy
+# -------------------------------
+if ! command -v trivy &> /dev/null; then
+    echo "🔧 Installing Trivy..."
+    sudo apt-get install -y wget apt-transport-https gnupg lsb-release
+
+    wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | sudo apt-key add -
+    echo deb https://aquasecurity.github.io/trivy-repo/deb $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/trivy.list
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)